scene/resources/primitive_meshes.cpp

```diff
@@ -1030,7 +1030,7 @@ void PlaneMesh::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::VECTOR2, "size", PROPERTY_HINT_NONE, "suffix:m"), "set_size", "get_size");
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "subdivide_width", PROPERTY_HINT_RANGE, "0,100,1,or_greater"), "set_subdivide_width", "get_subdivide_width");
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "subdivide_depth", PROPERTY_HINT_RANGE, "0,100,1,or_greater"), "set_subdivide_depth", "get_subdivide_depth");
-	ADD_PROPERTY(PropertyInfo(Variant::VECTOR3, "center_offset"), "set_center_offset", "get_center_offset");
+	ADD_PROPERTY(PropertyInfo(Variant::VECTOR3, "center_offset", PROPERTY_HINT_NONE, "suffix:m"), "set_center_offset", "get_center_offset");
```

Comment:
Primitive meshes can be used with MeshInstance2D, although they're primarily useful for 3D. So I'm not sure if they should actually have the suffix.

It looks like I missed this fact in PR too, several of these primitive meshes have `suffix:m` thanks to my PR, but I don't know if they should. I guess we can leave this up for discussion.

Reply:
Ok, sounds good. I think we should keep them too, because primitive meshes are almost always used for 3D whereas 2D meshes are used much more rarely.